app/views/upload.scala.html

```diff
@@ -1,6 +1,34 @@
-@()(implicit messages: Messages)
+    @()(implicit messages: Messages)
 
 @main(Messages("upload.header")) {
-    <h1 class="govuk-heading-xl">@Messages("upload.title")</h1>
-    <p>Placeholder page for upload</p>
-}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        @progressIndicator(Messages("progress.uploadStep"))
```

Comment:
Does this need to be updated to match the new message name?

Reply:
Apologies, unaware of how many minor mistakes I'm making today 🤦‍♀️